File: test/export/vb/08-operadores.vb
Module SIN_TITULO
	Sub Main()
		Dim a As Double
		Dim b As Double
		Dim c As Boolean
		Dim d As Boolean
		Dim e As String
		Dim f As String
		a = 11
		b = 2
		c = True
		d = False
		e = "Hola"
		f = "Mundo"
		Console.WriteLine(a+b)
		Console.WriteLine(a*b)
		Console.WriteLine(a/b)
		Console.WriteLine(a-b)
		Console.WriteLine(a^b)
		Console.WriteLine(a Mod b)
		Console.WriteLine(a=b)
		Console.WriteLine(a<>b)
		Console.WriteLine(a<b)
		Console.WriteLine(a>b)
		Console.WriteLine(a<=b)
		Console.WriteLine(a>=b)
		Console.WriteLine(c And d)
		Console.WriteLine(c Or d)
		Console.WriteLine(Not (c And d))
		Console.WriteLine(e&" "&f)
		Console.WriteLine(e.Equals(f))
		Console.WriteLine(Not e.Equals(f))
		Console.WriteLine(e.CompareTo(f)<0)
		Console.WriteLine(e.CompareTo(f)>0)
		Console.WriteLine(e.CompareTo(f)<=0)
		Console.WriteLine(e.CompareTo(f)>=0)
	End Sub
End Module

File: test/export/cpp/08-operadores.cpp
#include<iostream>
using namespace std;
int main() {
	float a;
	float b;
	bool c;
	bool d;
	string e;
	string f;
	a = 11;
	b = 2;
	c = true;
	d = false;
	e = "Hola";
	f = "Mundo";
	cout << a+b << endl;
	cout << a*b << endl;
	cout << a/b << endl;
	cout << a-b << endl;
	cout << pow(a,b) << endl;
	cout << a%b << endl;
	cout << a==b << endl;
	cout << a!=b << endl;
	cout << a<b << endl;
	cout << a>b << endl;
	cout << a<=b << endl;
	cout << a>=b << endl;
	cout << c && d << endl;
	cout << c || d << endl;
	cout << !(c && d) << endl;
	cout << e+" "+f << endl;
	cout << e==f << endl;
	cout << e!=f << endl;
	cout << e<f << endl;
	cout << e>f << endl;
	cout << e<=f << endl;
	cout << e>=f << endl;
	return 0;
}
